uportal-war/src/main/resources/layout/theme/respondr/navigation.xsl

```diff
@@ -105,19 +106,45 @@
     <xsl:param name="CONTEXT"/>  <!-- Catches the context parameter to know how to render the navigation. -->
     <chunk-point/> <!-- Performance Optimization, see ChunkPointPlaceholderEventSource -->
       <nav class="portal-nav">
-        <div class="container-fluid">
-        <a href="#" class="menu-toggle"><i class="fa fa-align-justify"></i> Menu</a>
+        <div id="sidebar">
```

Comment:
Does this "sidebar" interact well with the sidebar-left & sidebar-right regions?